cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/util/HttpUrlManager.kt

```diff
@@ -53,6 +55,16 @@ internal object HttpUrlManagerSingleton : HttpUrlManager {
             .build()
     }
 
+    override fun deleteCustomConsentToUrl(env: Env, params: CustomConsentReq): HttpUrl {
+        // https://cdn.privacy-mgmt.com/consent/tcfv2/consent/v3/custom/:propertyId?consentUUID={GDPR_UUID}
```

Comment:
@carmelo-iriti is it safe to leave exposed links like this in the repo?

Reply:
@Nevazhnovu that's a good question, I'll discuss it with Mike but I guess it's ok since all our APIs are open.

Reply:
@andresilveirah I guess it's okay since we use links in tests anyway